site/src/components/site-header.md

```diff
@@ -0,0 +1,499 @@
+---
+title: Site header
+description: The Site header component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - site-header
+---
+
+# Site header
+
+Site header component allows adding full width container to hold everything site's header might need - the logo, brand link, navigation, and other controls.
+
+**This page is best viewed with the content section full screened because banners are intended to be full width elements. Go full screen by clicking the button in the top right corner.**
+
+## Usage
+
+The site header is composed of the following elements:
+
+| Component             | Description                                                 |
+| --------------------- | ----------------------------------------------------------- |
+| `site-header`         | The main container for the site header.                     |
+| `site-header-button`  | Buttons within the site header.                             |
```

Comment:
Order here is a bit off, perhaps should be organized by DOM order and hierarchy?

1. site-header (parent)
1. site-header-logo
1. site-header-divider
1. site-header-nav
1. site-header-button

Reply:
Updated